internal/kafka/internal/config/kafka.go

```diff
@@ -86,3 +86,23 @@ func (c *KafkaConfig) ReadFiles() error {
 func (c *KafkaConfig) Validate(env *environments.Env) error {
 	return c.SupportedInstanceTypes.Configuration.validate()
 }
+
+func (c *KafkaConfig) GetFirstAvailableSize(instanceType string) (string, error) {
+	kafkaInstanceType, err := c.SupportedInstanceTypes.Configuration.GetKafkaInstanceTypeByID(instanceType)
+	if err != nil {
+		return "", err
+	}
+	return kafkaInstanceType.Sizes[0].Id, nil
+}
+
+func (c *KafkaConfig) GetKafkaInstanceSize(instanceType, sizeId string) (KafkaInstanceSize, error) {
```

Comment:
I believe this should return a `*KafkaInstanceSize` to avoid copying the object too many times